src/main/js/electron/index.py

```diff
@@ -1,79 +1,97 @@
+    end_count =request.form['to_count']
+    
     if not text:
       return render_template("welcome.html");
+
     text="datasets/"+text
     light_curve = pkg_resources.resource_stream(__name__,text)
     data = np.loadtxt(light_curve)
     Time = data[0:len(data),0]
     Rate = data[0:len(data),1]
-    Error= data[0:len(data),2 ]
-    error = np.array(Rate) - np.array(Rate)+50
-    Final_error = np.array(error).tolist();
-    #print(Final_error)
-    graphs = [     
-      dict(     
-          data=[
-              go.Scatter(
-               x=Time,
-               y=Rate,
-               error_y=dict(
+    Error= data[0:len(data),2]
+    
+
+    import sys
+    import os
+    from plotly import session, tools, utils
+    import uuid
+    import json
+
+   
+
+    trace1 = dict(
+            type = 'scatter',
+            x=Time,
+            y=Rate,
+            error_y=dict(
                type='data',
                array=Error,
                visible=True
                 )
-               
-             )
-         ],
-        layout=dict(
-                title='',
-                xaxis=dict(
+
+    )
+    
+    layout=dict(
+                 title='',
+                 xaxis=dict(
                      title='Time',
+                     range=[start_time,end_time],
+                     rangeslider=dict(),
                      titlefont=dict(
                      family='Courier New, monospace',
                      size=18,
                      color='#7f7f7f'
-                    )
+                        )
                  ),
-                yaxis=dict(
-                    title='Count Rate',
-                    titlefont=dict(
-                    family='Courier New, monospace',
-                    size=18,
-                    color='#7f7f7f'
-
-                  )
-                 ),
-                hovermode = 'closest'
-          
-                
-            )
-     )
-    ]
-    #print("yes")
-    #print(plotly.__version__)
-    ids = ['graph-{}'.format(i) for i, _ in enumerate(graphs)]
-    graphJSON = json.dumps(graphs, cls=plotly.utils.PlotlyJSONEncoder)
-    #print(graphJSON)
-    return render_template('index.html',
-                           ids=ids, 
-                           graphJSON=graphJSON)
+                 yaxis=dict(
+                     title='Count Rate',
+                     range=[start_count,end_count],
+                     titlefont=dict(
+                     family='Courier New, monospace',
+                     size=18,
+                     #range=[int(start), int(end)],
+                     #range=[10,20],
+                     color='#7f7f7f'
+                      )
+                 )
+        )
+    print("Here i am right")
+    fig = dict(data = [trace1], layout = layout)
+    plt.plot(fig, filename='templates/basic__plot.html',auto_open=False)
+
+    return render_template('index.html')
+
+
+
+
+
+@app.route('/')
+def my_form():
+    print("I am here")
```

Comment:
OK I see what you want to do with the debug print statements. The solution to your problem is logging: https://docs.python.org/3/library/logging.html
Output like this you would generate uising log.debug("enter my_form()") or something. For the end user we include a logging configuration file that the disables log statements for debug, but it something goes really wrong you may put log.critical("no output data - it seems the algorithm failed") or so, and this would be printed always (i..e for the developers as well as for the end users).